contracts/adapters/ProtocolAdapter.sol

```diff
@@ -158,6 +158,24 @@ library ProtocolAdapter {
         require(success, "exercise delegatecall failed");
     }
 
+    function delegateClaimRewards(
+        IProtocolAdapter adapter,
+        address rewardsAddress,
+        uint256[] calldata optionIDs
+    ) external returns (uint256) {
+        (bool success, bytes memory result) =
+            address(adapter).delegatecall(
+                abi.encodeWithSignature(
+                    "claimRewards(address,uint256[])",
+                    rewardsAddress,
+                    optionIDs
+                )
+            );
+        require(success, getRevertMsg(result));
+        require(success, "claimRewards delegatecall failed");
```

Comment:
Some duplication here

Reply:
Just noticed that it's the same across the whole ProtocolAdapter 😄 it's okay we can just address this line for now